blazor/3D-chart/accessibility.md

```diff
@@ -0,0 +1,93 @@
+
+<style>
+    .post .post-content img {
+        display: inline-block;
+        margin: 0.5em 0;
+    }
+</style>
+
+<div><img src="https://cdn.syncfusion.com/content/images/documentation/full.png" alt="Yes"> - All features of the component meet the requirement.</div>
+
+<div><img src="https://cdn.syncfusion.com/content/images/documentation/partial.png" alt="Intermediate"> - Some features of the component do not meet the requirement.</div>
+
+<div><img src="https://cdn.syncfusion.com/content/images/documentation/not-supported.png" alt="No"> - The component does not meet the requirement.</div>
+
+## WAI-ARIA attributes
+
+WAI-ARIA(Accessibility Initiative - Accessible Rich Internet Applications) defines a way to increase the accessibility of web pages, dynamic content, and user interface components developed with AJAX, HTML, JavaScript, and related technologies. ARIA provides additional semantics to describe the role, state, and functionality of web components.
+
+Element |Default description
+-----|-----
+Datalabel |Reads the Point y value.
+Legend |Click to show or hide the series.
+Axis Title |Reads the axis title.
+Chart Title |Reads the chart title.
+Series Points |Reads the Point x: Point y value.
+
+The Blazor 3D Chart component followed the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns to meet the accessibility. The following ARIA attributes are used in the Blazor 3D Chart component:
+
+* img (role)
+* button (role)
+* region (role)
+* aria-label (attribute)
+* aria-hidden (attribute)
+* aria-pressed (attribute)
+
+## Keyboard navigation
+
+The Blazor 3D Chart component followed the [keyboard interaction](https://www.w3.org/WAI/ARIA/apg/patterns/alert/#keyboardinteraction) guideline, making it easy for people who use assistive technologies (AT) and those who completely rely on keyboard navigation. The following keyboard shortcuts are supported by the Blazor 3D Chart component.
+
+Interaction Keys |Description
+-----|-----
+<kbd>Alt + J</kbd> | Moves the focus to the chart element.
+<kbd>Tab</kbd> |Moves the focus to the next element in the chart.
+<kbd>Shift + Tab</kbd> |Moves the focus to the previous element in the chart.
+<kbd>DownArrow</kbd> |Moves the focus to the data point right side from the selected point.
+<kbd>UpArrow</kbd> |Moves the focus to the data point right side from the selected point.
+<kbd>Left Arrow</kbd> |Moves the focus to the next series in our Chart component.
+<kbd>Right Arrow</kbd> |Moves the focus to the previous series in our Chart component.
+<kbd>Enter/Space</kbd> |Selects the data point in the series
+<kbd>Down/Left Arrow</kbd> |Moves the focus to the legend left side from the selected legend.
+<kbd>Up/Right Arrow</kbd> | Moves the focus to the legend right side from the selected legend.
+<kbd>Enter/Space</kbd> |Toggles the visibility of the corresponding series.
+<kbd>Ctrl + +</kbd> |Zoom in the chart.
```

Comment:
Zooming is not applicable for 3D Chart, can you check and remove @IlamathiSF3875